app/views/characters/edit.haml

```diff
@@ -12,32 +12,24 @@
 
 = form_for @character, url: @character, method: :put do |f|
   .character-form
-    %table.form-table
-      %thead
-        %tr
-          %th.editor-title{colspan: 2}= @character.name
+    .form-table#character-editor
+      .editor-title= @character.name
       = render 'editor', f: f
 
     %br
-    %table.form-table
-      %thead
-        %tr
-          %th.editor-title
-            Aliases and Pseudonyms
-            - if @character.user_id == current_user.id
-              = link_to new_character_alias_path(@character) do
-                .link-box.action-new + New Alias
-      %tbody
-        - @aliases.each do |calias|
-          %tr
-            %td.padding-5{class: cycle('even', 'odd')}
-              = calias.name
-              - if @character.user_id == current_user.id
-                .float-right
-                  = link_to character_alias_path(@character, calias), method: :delete, confirm: 'Are you sure you want to delete this alias?' do
-                    = image_tag "icons/cross.png", alt: 'Delete'
-        - if @aliases.empty?
-          %tr
-            %td.centered.padding-10{ class: cycle('even', 'odd'), colspan: 6 } — No aliases yet —
+    .form-table
+      .editor-title
+        Aliases and Pseudonyms
+        - if @character.user_id == current_user.id
+          = link_to new_character_alias_path(@character) do
+            .link-box.action-new + New Alias
```

Comment:
![image](https://github.com/user-attachments/assets/57e77fbd-818a-435f-9e21-938e4852835b)
![image](https://github.com/user-attachments/assets/4c76757b-0cc2-4d32-b0e8-69511960ba65)

The "+ New Alias" button got misaligned with the title, we should probably fix that.